contrib/completion/bash/docker

```diff
@@ -933,7 +933,7 @@ _docker_images() {
 
 	case "$cur" in
 		-*)
-			COMPREPLY=( $( compgen -W "--all -a --digests --filter -f --help --no-trunc --quiet -q" -- "$cur" ) )
+			COMPREPLY=( $( compgen -W "--all -a --digests --filter -f --format --help --no-trunc --quiet -q" -- "$cur" ) )
```

Comment:
As `--format` takes an argument, completion also has to take care of that, see https://github.com/docker/docker/blob/master/contrib/completion/bash/docker#L984-L987 for an example.